composer.json

```diff
@@ -31,7 +31,7 @@
     "source": "https://github.com/browscap/browscap"
   },
   "require": {
-    "php": ">= 5.5.0",
+    "php": ">= 5.6.0",
```

Comment:
The change to min version should ideally be done in a separate PR but I'll accept in this case :|